src/VisualStudio/Core/Def/ProjectSystem/VisualStudioWorkspaceImpl_SourceGenerators.cs

```diff
@@ -70,9 +69,8 @@ public void SubscribeToSourceGeneratorImpactingEvents()
     }
 
     [Export(typeof(ICommandHandler))]
-    [ContentType(ContentTypeNames.RoslynContentType)]
-    [ContentType(ContentTypeNames.XamlContentType)]
     [Name(PredefinedCommandHandlerNames.SourceGeneratorSave)]
+    [ContentType(StandardContentTypeNames.Text)]
```

Comment:
This feels like its going to cause a lot of RPS regressions as this will now run on any content type.  Can you do an RPS validation to check?